ui/components/component-library/avatar-token/avatar-token.stories.tsx

```diff
@@ -324,7 +323,9 @@ ShowHalo.args = {
   showHalo: true,
 };
 
-export const ColorBackgroundColorAndBorderColor = (args) => (
+export const ColorBackgroundColorAndBorderColor: ComponentStory<
+  typeof AvatarToken
+> = (args) => (
```

Comment:
nit: does this need to be on a new line or was it automatically formatted that way by eslint?
```suggestion
export const ColorBackgroundColorAndBorderColor: ComponentStory<typeof AvatarToken> = (args) => (
```

Reply:
It was automatically formatted by lint:eslint:fix